tests/test_client.py

```diff
@@ -55,4 +64,24 @@ def test_delete_catalog(self, client, requests_mock):
         """Test delete_catalog method"""
         requests_mock.delete(data_url)
         r = client.delete_catalog(catalogID)
+        assert r is None
+
+    # test fdp
+    def test_create_fdp(self, client, data_fdp, requests_mock):
```

Comment:
Not related to this PR, but what does `create_fdp` do? You cannot really create an FDP from the client side, can you?

Reply:
It's creating the the metadata of the `fdp layer`. When initiating the FDP backend (our implementation), the `BASE_URL/fdp` is empty, you have to  add some metadata to it.